packages/ui-view/src/ContextView/index.tsx

```diff
@@ -168,12 +168,10 @@ class ContextView extends Component<Props & OtherHTMLAttributes<Props>> {
   }
 
   componentDidMount() {
-    // @ts-expect-error ts-migrate(2722) FIXME: Cannot invoke an object which is possibly 'undefin... Remove this comment to see the full error message
-    this.props.makeStyles()
+    this.props.makeStyles?.()
   }
   componentDidUpdate() {
-    // @ts-expect-error ts-migrate(2722) FIXME: Cannot invoke an object which is possibly 'undefin... Remove this comment to see the full error message
-    this.props.makeStyles()
+    this.props.makeStyles?.()
```

Comment:
Is this the same thing as `this.props?.makeStyles()`

Reply:
To be honest, I'm not sure.
According to the [docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#optional_chaining_with_function_calls) this the correct way of using optional chaining if we want to call an optional method property.